### PR TITLE
cups 2.4.0

### PR DESCRIPTION
**Pull request by bayandin**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/cups.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class Cups < Formula
   desc "Common UNIX Printing System"
   homepage "https://github.com/OpenPrinting/cups"
-  url "https://github.com/OpenPrinting/cups/releases/download/v2.3.3op2/cups-2.3.3op2-source.tar.gz"
+  url "https://github.com/OpenPrinting/cups/releases/download/v2.4.0/cups-2.4.0-source.tar.gz"
   # This is the author's fork of CUPS. Debian have switched to this fork:
   # https://lists.debian.org/debian-printing/2020/12/msg00006.html
```
Comment (MusbahuTijjani) [inline Formula/cups.rb:6]:
```suggestion
  # https://lists.debian.org/debian-printing/2020/12/msg00006.html
```

**Comment by carlocab** [comment]
```
configure: error: --with-tls=yes was specified but no compatible TLS libraries could be found.
```

**Comment by carlocab** [comment]
```
checking for libgnutls-config... no
  configure: error: --with-tls=yes was specified but no compatible TLS libraries could be found.
```

I think you need `gnutls`.

**Review by iMichka — APPROVED** [review]
I had a look at the Linux issues. Most of them are already known.
I opened a few PRs to fix some of the others

> amdatu-bootstrap -> might need a rev bump
apache-drill -> might need a rev bump
apache-flink -> might need a rev bump
apache-geode -> might need a rev bump
commandbox -> might need a rev bump
davmail -> might need a rev bump
druid #90196
emscripten -> linkage to llvm
gradle-profiler -> might need a rev bump
infer -> old known issue, does not build with xccode 12/13
javarepl -> issue already seen in the last openjdk PR
jolie -> issue already seen in the last openjdk PR
jruby -> conflict with ruby
ktlint -> might need a rev bump
maxwell -> might need a rev bump
mrbayes -> issue already seen in the last openjdk PR
parquet-cli -> issue already seen in the last openjdk PR
redpen -> might need a rev bump
rest-shell -> might need a rev bump
rhino #90312
solr@7.7 -> might need a rev bump, can't download sources anymore
swagger2markup-cli #90311
xmlsh -> might need a rev bump, can't download sources anymore

I will slowly work through the list later. Let's ship this as the failures are unrelated.

**Pull request closed without merging**